Use generated files in module test

lucylq opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #10497

Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

lucylq committed:
Use generated files in module test

Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

JacobSzwejbka reviewed (approved):
thanks!

shoumikhin commented on extension/module/test/module_test.cpp:
```diff
@@ -26,9 +27,9 @@ class ModuleTest : public ::testing::Test {
     if (const char* env = std::getenv("RESOURCES_PATH")) {
       resources_path = env;
```
Is this path used for anything anymore?

lucylq replied:
ah it's not. I'll remove it, thanks

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]

lucylq committed:
Update on "Use generated files in module test"


Instead of checking in the PTE/PTD files. This helps us keep up to date with aot changes, especially with incoming flat_tensor refactor.

Main changes here are with the model. Instead of using AddModule from examples/portable, use ModuleAdd from test/models/export_program, like our other tests. There is a shape difference (1 tensor vs 2 tensors+alpha), so we have to change all the inputs.

Differential Revision: [D73470865](https://our.internmc.facebook.com/intern/diff/D73470865/)

[ghstack-poisoned]
